pkg/plugins/adapters/adapters.go

```diff
@@ -28,6 +28,7 @@ func ModelToInstanceSettings(ds *datasources.DataSource, decryptFn func(ds *data
 	}
 
 	return &backend.DataSourceInstanceSettings{
+		Type:                    ds.Type,
```

Comment:
@gabor i think this change was lost at some point, but it's working locally for me now after adding it back

Reply:
@toddtreece nice find! i tested it with Loki, and it does work fine.